systemvm/debian/lib/systemd/system/open-vm-tools.service

```diff
@@ -0,0 +1,12 @@
+﻿[Unit]
+Description=Service for virtual machines hosted on VMware
+Documentation=http://open-vm-tools.sourceforge.net/about.php
+DefaultDependencies=no
+Before=cloud-init-local.service
```

Comment:
@PaulAngus I see references to cloud-init-local.service and cloud-init.service but they are not in codebase or the template. Is there any file to be added, or did you mean cloud-early-config.service?